src/Criteo.OpenApi.Comparator.Cli/Program.cs

```diff
@@ -45,15 +47,41 @@ public static int Main(string[] args)
         }
 
         private static bool TryReadFile(string path, out string fileContent)
+        {
+            bool readOk = TryReadDistantFile(path, out fileContent);
+            if (!readOk)
+            {
+                TryReadLocalFile(path, out fileContent);
```

Comment:
It would be beneficial to add a null check for `fileContent` after trying to read from both a distant URL and a local file. This can prevent potential NullReferenceExceptions in the future. [medium]